frontend/e2e/database/db.ts

```diff
@@ -0,0 +1,38 @@
+import pg from "pg"
+
+export default class DB {
+  private DBConfig = {
+    user: "test",
```

Comment:
Nitpick: I know these credentials are just our local DB stuff and most likely going to be always the same but it still feels like this should be in the env. Not a required change though, just something to consider.

Reply:
I generally aggree but since we have this information already in our docker-compose files and it can't be misused I'd leave it like that for now.